Source/Core/DiscIO/VolumeVerifier.cpp

```diff
@@ -1070,10 +1069,19 @@ void VolumeVerifier::WaitForAsyncOperations() const
 bool VolumeVerifier::ReadChunkAndWaitForAsyncOperations(u64 bytes_to_read)
 {
   std::vector<u8> data(bytes_to_read);
+
+  const u64 bytes_to_copy = std::min(m_excess_bytes, bytes_to_read);
+  if (bytes_to_copy > 0)
+    std::memcpy(data.data(), m_data.data() + m_data.size() - m_excess_bytes, bytes_to_copy);
```

Comment:
Can destination and source overlap here?

Reply:
No. The source is the class member `m_data` and the destination is the local variable `data`.

Reply:
Bliiiiiiiind.

Reply:
If it makes you feel any better, I originally wrote this commit in August and couldn't be bothered debugging a critical problem with it until today, and it turned out that part of the problem was that I had written `m_data` for the destination instead of `data` :)